source/usage-examples/findOneAndDelete.txt

```diff
@@ -0,0 +1,81 @@
+===================
+Delete One Document
+===================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can delete a single document in a collection with
+``collection.findOneAndDelete()``.
+The ``findOneAndDelete()`` method uses a query document that you provide
+to match only the subset of the documents in the collection that match
+the query. If you don't provide a query document (or if you provide an
+empty document), MongoDB matches all documents in the collection.
+However, only the first matched document is deleted.
+
+You can also define additional query options such as
+`sort <https://docs.mongodb.com/manual/reference/bson-type-comparison-order/index.html>`_
+and
+`projection <https://docs.mongodb.com/manual/tutorial/project-fields-from-query-results/>`_
+to configure the returned document. You should specify these options
+through the ``options`` object passed as the second parameter of the
+``findOneAndDelete`` method. For detailed reference documentation, see
```

Comment:
It seems there is some inconsistency around method naming. In some places you keep the parentheses, elsewhere like here you discard them. Can we make that consistent? i.e. `findOneAndDelete()` vs. `findOneAndDelete`